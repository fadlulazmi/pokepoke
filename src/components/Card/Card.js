import React from 'react';
import './styles.scoped.css'

export default function Card(props) {
  const { counter, name } = props;
  return (
    <div className="card">
      <p>{name}</p>
      <p>Owned: {counter}</p>
    </div>
  );
}
