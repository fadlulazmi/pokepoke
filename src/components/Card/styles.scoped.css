.card {
  border: lightgray 1px solid;
  width: 140px;
  height: 100px;
  margin: 10px;
  display: flex;
  flex-direction: column;
  place-content: center;
  align-items: center;
  border-radius: 10px;
  background-color: rgba(255, 255, 255, .2);
  text-decoration: none;
  transition: 1s;
}

.card > p {
  background-color: rgba(255, 255, 255, 0);
  color: azure;
  font-weight: 500;
}

.card > div {
  background-color: rgba(255, 255, 255, 0);
  color: azure;
  font-weight: 500;
  display: flex;
  align-items: center;
  justify-content: space-around;
  width: 100px
}

.card > .zero > p {
  transition: 1s;
  background-color: rgba(255, 255, 255, 0);
  color: pink;
}

.card > .active > p {
  transition: 1s;
  background-color: rgba(255, 255, 255, 0);
  color: greenyellow;
}