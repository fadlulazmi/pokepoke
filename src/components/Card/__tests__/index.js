import Card from '../Card'
import ShallowRenderer from 'react-test-renderer/shallow';


describe('src/components/Card', () => {
  test('should be rendered correctly', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<Card />);
    expect(tree).toMatchSnapshot();
  })
})
