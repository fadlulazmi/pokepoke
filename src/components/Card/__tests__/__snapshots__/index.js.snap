// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/Card should be rendered correctly 1`] = `
<div
  className="card"
>
  <p />
  <div
    className="active"
  >
    Owned: 
    <p />
  </div>
</div>
`;
